### DISCUSSION TITLE
Does NetKVM driver can be used on Host OS(PF device)?

### DISCUSSION THREAD

**Discussion by thasreefkhalith** [virtio-win/kvm-guest-drivers-windows#908] [Q&A]
I wanted check whether NetKVM driver can be used on PF device

**Comment by YanVugenfirer** [comment]
Hi,

Yes, it can. At least several vendors tested it.
But it doesn’t support currently SR-IOv (so you won’t be able to use VFs). Supporting SR-IOv is in our roadmap.

**Comment by YanVugenfirer** [comment]
If you encounter some issues, please report them.
